packages/epo-react-lib/src/layout/BasicModal/BasicModal.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  ComponentMeta,
+  ComponentStoryObj,
+  ComponentStory,
+} from "@storybook/react";
+import { objChildren } from "@/storybook/utilities/argTypes";
+
+import BasicModal from ".";
+import Button from "@/atomic/Button";
+import { useState } from "react";
+
+const meta: ComponentMeta<typeof BasicModal> = {
+  component: BasicModal,
+  argTypes: {
+    children: objChildren,
+    open: {
+      type: "boolean",
+      description: "Determines if the modal is visible or not.",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    darkMode: {
+      type: "boolean",
+      description: "Sets the dark mode theme.",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    title: {
+      type: "string",
+      description: "Modal title displayed at the top.",
+      table: {
+        type: {
+          summary: "string",
+        },
+      },
+    },
+    description: {
+      type: "string",
+      description: "Modal description displayed after the title.",
+      table: {
+        type: {
+          summary: "string",
+        },
+      },
+    },
+    onClose: {
+      type: "function",
+      action: "Closed",
+      description:
+        "Callback either when user clicks outside the modal or clicks the close button. Close button will attach an event, click outside will not.",
+      table: {
+        type: {
+          summary: "(event?: MouseEvent) => void",
+        },
+      },
+    },
+  },
+};
+export default meta;
+
+const Template: ComponentStory<typeof BasicModal> = ({
+  open,
+  title,
+  description,
+  darkMode,
+  children,
+}) => {
+  const [isOpen, setIsOpen] = useState(open || false);
+
+  return (
+    <>
+      <Button onClick={() => setIsOpen(true)}>Click to open modal</Button>
+      <BasicModal
```

Comment:
modals are initiated closed and opened using this template component, otherwise in the Docs page the modal takes over the entire screen and can't be closed.